cli/src/main/resources/scalaxb.scala.template

```diff
@@ -821,14 +821,13 @@ class HexBinary(_vector: Vector[Byte]) extends scala.collection.IndexedSeq[Byte]
 
 object HexBinary {
   def apply(xs: Byte*): HexBinary = {
-    import scala.collection.breakOut
-    val vector: Vector[Byte] = (xs.toIndexedSeq map {x: Byte => x})(breakOut)
+    val vector: Vector[Byte] = xs.toVector
     new HexBinary(vector)
   }
 
   def apply(value: String): HexBinary = {
     val array = DatatypeConverter.parseHexBinary(value)
-    apply(array: _*)
+    apply(array.toSeq: _*)
```

Comment:
fixes warning `Passing an explicit array value to a Scala varargs method is deprecated (since 2.13.0) and will result in a defensive copy; Use the more efficient non-copying ArraySeq.unsafeWrapArray or an explicit toIndexedSeq call`